### PR TITLE
Cython extension for rmm::cuda_stream_pool

### PR DESCRIPTION
**Pull request by divyegala**

### CODE REVIEW

**Comment by divyegala** [comment]
@jakirkham could I get a review from you for the cython side? Also @jrhemstad, I have removed `explict` from the conversion operator between `rmm::cuda_stream_view` and `cudaStream_t`

File: include/rmm/cuda_stream_view.hpp [hunk 1]
```diff
@@ -59,7 +59,7 @@ class cuda_stream_view {
   /**
    * @brief Explicit conversion to cudaStream_t.
```
Comment (harrism) [inline include/rmm/cuda_stream_view.hpp:60]:
```suggestion
   * @brief Implicit conversion to cudaStream_t.
```

**Review by harrism — CHANGES_REQUESTED** [review]
Just need to update the brief.

**Review by harrism — APPROVED** [review]

**Review by galipremsagar — APPROVED** [review]

**Review by shwina — APPROVED** [review]

**Comment by harrism** [comment]
@gpucibot merge

**Pull request merged**